Backend/offers/offersRoutes.js

```diff
@@ -0,0 +1,31 @@
+const offersController = require("./offersController");
+const offersValidator = require("./offersHelper");
+const checkLoggedIn = require("../middlewares/restrictedMiddleware");
+const express = require("express");
+
+const router = express.Router();
+
+router.post(
+  "/",
+  checkLoggedIn,
+  offersValidator.validateOfferBody,
+  offersController.createOffer
+);
+
+router.get("/all-offers", offersController.getAllOffers);
+router.get("/:id([0-9]+)", checkLoggedIn, offersController.getMyOffers);
```

Comment:
Why is it called getMyOffers, by we still supply an id? Maybe it should be getOffersByMakerId and supply make route like `/user/:id/offers`?

Reply:
Check

Reply:
router.get("/user/:id/offers", checkLoggedIn, offersController.getOffersByMakerId);